integrators/ordinary_differential_equations.hpp

```diff
@@ -116,8 +119,8 @@ class FixedStepSizeIntegrator : public Integrator<DifferentialEquation> {
   // ]problem.t_final - step, problem.t_final].
   // |problem.append_state| will be called with |state.time.values|s at
   // intervals differing from |step| by at most one ULP.
-  virtual void Solve(IntegrationProblem<ODE> const& problem,
-                     Time const& step) const = 0;
+  virtual Status Solve(IntegrationProblem<ODE> const& problem,
+                       Time const& step) const = 0;
```

Comment:
No implementations return anything other than `OK`. For the moment, keep it `void`, we'll use `Status`es if we ever want early termination of fixed-step integrations.